debian/copyright

```diff
@@ -0,0 +1,12 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: pop-launcher
```

Comment:
The contents of this file seem to be generally inaccurate. The project name/url is wrong, and the license for `Files: *` is wrong.